README.md

```diff
@@ -61,26 +71,23 @@ optional arguments:
                         Model types: "dense-unet", "unet" (default: "dense-
                         unet")
   --model_wts_path MODEL_WTS_PATH
-                        Path for model wts to be used (default='./saved_model_
-                        wts/dense_unet_back2front_finetuned.pth')
-  --is_mgz [IS_MGZ]     Is the image in .mgz format (default=False, default
-                        format is .nii.gz)
-  --save_prob [SAVE_PROB]
-                        Should the softmax prob values for each voxel be saved
-                        ? (default: False)
-  --use_gpu [USE_GPU]   Use GPU for inference? (default: True)
+                        Path for model wts to be used, provide a model from
+                        saved_model_wts/
+  --is_mgz              Use this flag when image is in .mgz format
+  --save_prob           Use this flag to save the softmax prob values for each
+                        voxel
+  --use_gpu             Enable GPU usage (CPU is used by default)
 
 ```
 An example could look something like this:
+
 ```
-python3 perform_pred.py --input_image_path './../../../data_orig/199251/mri/T1.mgz' \
+perform_pred.py --input_image_path './../../../data_orig/199251/mri/T1.mgz' \
 --segmentation_dir_path './sample_pred/' \
```

Comment:
Based on my `setup.py` already putting `perform_pred.py` in user PATH, specification of `python3` interpreter isn't necessary.